docs/dinamicko-programiranje/knapsack.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Knapsack
+---
+
+### Problem
+
+Ruksak kapaciteta $x$ moramo napuniti sa stvarima kojih ima $n$, a od kojih svaka zauzima određeni kapacitet i ima određenu vrijednost. Naravno cilj nam je maksimizirati ukupnu vrijednost stvari u ruksaku. U zadatku bitnu razliku čini jesu li stvari tekuće ili čvrste.
```

Comment:
Trebalo bi napisati što točno znači da je stvar tekuća.
Npr, tekuće stvari su one kojih se može uzeti dio, a čvrste one kojih se može uzeti samo cijela stvar ili ništa.
Engleski nazivi su fractional i 0/1 knapsack problem.

Reply:
Jel bitno napisat te engleske nazive

Reply:
nije